composer.json

```diff
@@ -20,6 +20,7 @@
         "mikey179/vfsstream": "^1.6",
         "sllh/composer-versions-check": "^2.0",
         "spryker-sdk/architector": "0.1.x-dev",
+        "spryker-sdk/async-api": "1.0.x-dev",
```

Comment:
what is it? library to work with xml files?
is it ok that it's under spryker-sdk?

Reply:
It is a library for parsing AsyncAPI files. Haven't found anything out there to use so I wrote my own one. IMO it is fine in spryker-sdk.